src/lib_ccx/ccx_sub_entry.pb_c.h

```diff
@@ -0,0 +1,86 @@
+/* Generated by the protocol buffer compiler.  DO NOT EDIT! */
```

Comment:
@cfsmp3,
1. It passes not all, but same tests as current master branch of ccextractor repo
2. Sure I will provide all documentation including changes. This is a "familiarization" branch for you all to try the feature. I've pulled a request only because you asked for it.